scripts/Obtain-Workspace.ps1

```diff
@@ -170,19 +164,23 @@ Write-Host
 
 Create-ComponentConfigJson($epoch_secs)
 
-Write-Host
-Write-Host "Checking for old resource groups"
-Write-Host
 
-$old_rg_list =Get-OldResourceGroups($epoch_secs-2*$window_seconds)
-if( $old_rg_list.count -gt 0)
-{
-    Write-Host "Found $($old_rg_list.count) resource groups to clean up"
-    foreach( $rg in $old_rg_list)
-    {
-        Write-Host "Cleaning up $($rg.name)"
-        az group delete --name $rg.name --yes
+
+Write-Host
+if ( $cullWorkspaces ) {
+    Write-Host "Checking for old resource groups ($cullWorkspaces)"
```

Comment:
its probably better to do this before workspace creation in case we're close to the 100 limit and need to free up space before creation